src/content/upgrades/merge/index.md

```diff
@@ -5,64 +5,236 @@ lang: en
+
+Let's consider an example. Imagine Ethereum is a spaceship that isn't quite ready for an interstellar voyage. With the Beacon Chain, the community has built a new engine and a hardened hull. After significant testing, it's almost time to hot-swap the new engine for the old mid-flight. This will merge the new, more efficient engine into the existing ship, ready to put in some serious lightyears and take on the universe.
+
+## Merging with Mainnet {#merging-with-mainnet}
+
+Since genesis, Mainnet has been secured by proof-of-work. This is the main Ethereum blockchain we're all used to—it contains every transaction, smart contract, and balance since it began in July 2015.
+
+During this growing history, developers have been hard at work preparing for an eventual transition away from proof-of-work to proof-of-stake instead. On December 1, 2020, the Beacon Chain was created, which has since existed as a separate blockchain to Mainnet, running in parallel.
+
+The Beacon Chain has not been processing Mainnet transactions, and for all intents and purposed has simply been coming to consensus on the state of itself... which is great and all, but after extensive testing, the time for it to come to consensus on more than just itself is rapidly approaching.
+
+The Merge itself represents the official switch to using the Beacon Chain as the official means of block production. Mining will no longer be the means of producing valid blocks, and instead this role will land with the proof-of-stake validators who will then be responsible for processing the validity of all transactions, and proposing blocks.
+
+No history is lost. As Mainnet is merged with the Beacon Chain, the entire transactional history of Ethereum comes with it. You don't need to do anything. Your funds are safe.
+
+<InfoBanner>
+This transition to proof-of-stake will come with some changes to the way ether is supplied. Learn more about <a href="/upgrades/merge/eth-issuance/">ether issuance before and after The Merge</a>.
+</InfoBanner>
+
+## What to I need to do to get ready? {#preparing-for-the-merge}
+
+The Merge is one of the most significant and anticipated upgrades in the history of Ethereum, and although in the long-term its impact will be felt by everyone, in the near-term some folks will need to take action to be fully prepared.
+
+For everyday users, holders, or non-node-operating stakers there is **nothing you need to do**, but a few things you should be aware of heading into The Merge. [More on this below.](#users-holders)
```

Comment:
I feel like this this should somehow be bold or a louder callout, as this is probably the bulk of readers.